tests/extract/test_extract_job.json

```diff
@@ -117,17 +126,7 @@
         "finished_line_number": 88739,
         "name": "Unnamed step",
         "result": 7,
-        "started_line_number": 0,
-        "text_log_error": [
-          {
-            "line": "line contents here",
-            "line_number": 619845839
-          },
-          {
-            "line": "ERROR! more line contents",
-            "line_number": 6
-          }
-        ]
```

Comment:
we will let the `text_log_error` be attached to the job instead of  `text_log_step`